contracts/staking/StakingPool.sol

```diff
@@ -58,6 +60,17 @@ abstract contract StakingPool {
         _;
     }
 
+    function initialize(StakingPoolInfo.StakingPoolData calldata info)
```

Comment:
nit; s/`StakingPoolData`/`Data`.

The contract already scope the data as being in a `StakingPoolInfo`, does repeating `StakingPool` again really add any extra information to the reader?